templates/home.html

```diff
@@ -23,29 +23,55 @@ <h2>Animals in the Shelter</h2>
             <form id="addAnimalForm">
                 <div class="form-group">
                     <label for="animal_type">Animal Type:</label>
-                    <input type="text" class="form-control" id="animal_type" placeholder="Animal Type" required>
+                    <select class="form-control" id="animal_type" required>
+                        <option value="Dog">Dog</option>
+                        <option value="Cat">Cat</option>
```

Comment:
If you want, you can implement better.
You can create StrEnum
https://docs.python.org/3/library/enum.html#enum.StrEnum
with Dog and Cat, and return it's members in context.
In this case you'll have defined types in backend instead of frontend.
I'm not saying that you implementation will not work, but I see how you trying to do the best, that's why suggesting such option